src/sickle.ts

```diff
@@ -36,12 +43,15 @@ class Annotator {
 
   constructor(private options: SickleOptions) { this.indent = 0; }
 
-  annotate(file: ts.SourceFile): string {
+  annotate(file: ts.SourceFile): SickleOutput {
     this.output = [];
     this.file = file;
     this.visit(file);
     this.assert(this.indent == 0, 'visit() failed to track nesting');
-    return this.output.join('');
+    return {
+      output: this.output.join(''),
+      externs: null,
+    };
```

Comment:
I always prefer the former, but I admit I didn't think about it too hard.  This is mostly consistent with how TypeScript does it as well (e.g. when we're compiling a file via the TS API, it gives us back an object with a few fields like this).